### ISSUE TITLE
thumbnails talks to webdav via proxy

### ISSUE THREAD

**Issue by wkloucek** [owncloud/ocis#1385]
## Describe the bug
The thumbnails extension talks to the webdav extension via the proxy.
This is also reflected in following configuration item (https://owncloud.github.io/extensions/thumbnails/configuration/):
```
-webdavsource-baseurl | $THUMBNAILS_WEBDAVSOURCE_BASEURL
Base url for a webdav api. Default: https://localhost:9200/remote.php/webdav/
```

## Steps to reproduce

1. start oCIS in default configuration

## Expected behavior
Thumbnails uses service discovery in order to talk to the webdav api directly.

## Actual behavior
Thumbnails talks to webdav api via the proxy. The proxy address is hardcoded.

## Setup

oCIS version: 1.0.0 Tech Preview
Default configuration

## Additional context
This creates various problems. Eg. if you turn off SSL termination of the proxy (`PROXY_TLS=false`), you must also change `THUMBNAILS_WEBDAVSOURCE_BASEURL` to `THUMBNAILS_WEBDAVSOURCE_BASEURL=http://localhost:9200/remote.php/webdav/` so that it doesn't use https.

**Comment by wkloucek** [comment]
@refs do you have an opinion on this? Can service discovery be used for this?

**Comment by C0rby** [comment]
> @refs do you have an opinion on this? Can service discovery be used for this?

Not exactly service discovery but yeah we could use the cs3 api provided by reva like we do in other services.

**Comment by wkloucek** [comment]
Looks like this was closed by #1898. It still routes via the proxy but isn't an hardcoded config option anymore.
